content/applications/sales/crm/optimize/gamification.rst

```diff
@@ -1,86 +1,159 @@
-==========================================
-How to motivate and reward my salespeople?
-==========================================
+================
+CRM Gamification
+================
 
-Challenging your employees to reach specific targets with goals and rewards is an excellent way to
-reinforce good habits and improve your salespeople productivity. The Gamification module gives you
-simple and creative ways to motivate and evaluate your employees with real-time recognition and
-badges inspired by game mechanics.
+In Odoo's *CRM* app, *Gamification tools* provide the opportunity to evaluate and motivate users
+through customizable challenges, goals, and rewards. Goals are created to target actions within the
+*CRM* app, and can be tracked and rewarded automatically.
 
 Configuration
 =============
 
-Install the *Gamification* module, or the *CRM gamification* one, which adds some useful data
-(goals and challenges) that can be used on *CRM/Sale*.
+To install the *Gamification* module, navigate to the :menuselection:`Apps` application. Click into
```

Comment:
On a follow-up PR let's re-examine this section. It's not exactly clear how or when these apps are installed automatically so let's try to cover that.